application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/pages/TlabPage.java

```diff
@@ -153,14 +211,21 @@ public TlabUI(Composite parent, FormToolkit toolkit, IPageContainer editor, ISta
 			methodsCT = new TlabChartTable(tabFolder, toolkit, editor, state, JdkAttributes.STACK_TRACE_TOP_METHOD);
 			DataPageToolkit.addTabItem(tabFolder, methodsCT.getComponent(), Messages.TlabPage_METHODS_TAB_NAME);
 
+			classCT = new TlabChartTable(tabFolder, toolkit, editor, state, JdkAttributes.ALLOCATION_CLASS);
+			DataPageToolkit.addTabItem(tabFolder, classCT.getComponent(), Messages.TlabPage_CLASS_TAB_NAME);
+
+			summaryCT = new TlabSummaryUI(tabFolder, toolkit, editor, state);
+			DataPageToolkit.addTabItem(tabFolder, summaryCT.getComponent(), Messages.TlabPage_SUMMARY_TAB_NAME);
+
 			tabFolder.setSelection(tabFolderIndex);
 		}
 
 		@Override
 		public void saveTo(IWritableState state) {
 			threadsCT.saveTo(state);
 			methodsCT.saveTo(state);
-
+			classCT.saveTo(state);
```

Comment:
This page looks fine to me as well, but it's a bit of a stealth update as there wasn't mention in the bug report or issue name regarding adding a class page.

Reply:
Yes. Customer was missing class page and summary page. Actually they both were present in JMC 5.5 and customer wanted it back. I have mentioned it in JIRA also.